docs/index.md

```diff
@@ -0,0 +1,14 @@
+# StarkNet Smart Contracts
+
+## Introduction
+
+The climate emergency is the greatest challenge of our time, and requires immediate & widespread action.  
+Blockchain technology is revolutionary and opens up a new world of opportunities.  
+What if we leveraged Blockchain as a tool to fight climate change?  
+
+Our smart contracts were first deployed on Cosmos. To scale up our business we recentely decided to move on StarkNet. A blockchain's sovereignty is a tool that allows a community to generate the conditions needed to solve financial and coordination issues.
```

Comment:
It's not necessary to tell that we where on Cosmos. It will bring more question that it's answer questions. 
And at the end, from a project perspective, we where a really short time on it. 

`Our smart contracts were first deployed on Cosmos. To scale up our business we recentely decided to move on StarkNet. A blockchain's sovereignty is a tool that allows a community to generate the conditions needed to solve financial and coordination issues.`

To 

`A blockchain's sovereignty is a tool that allows a community to generate the conditions needed to solve financial and coordination issues. For this reasons, to scale up our business we decided to build on StarkNet.`